src/Utilities/ElectronicSignatureModel.cs

```diff
@@ -0,0 +1,80 @@
+﻿//-------------------------------------------------------------
+// <copyright file="ElectronicSignatureModel.cs" company="GlobalLink Vasont">
+// Copyright (c) GlobalLink Vasont. All rights reserved.
+// </copyright>
+//-------------------------------------------------------------
+namespace Vasont.Inspire.Models.Utilities
+{
+    using System;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Converters;
+    using Vasont.Inspire.Models.Security;
+
+    /// <summary>
+    /// Contains an enumerated list of resource types.
+    /// </summary>
+    [JsonConverter(typeof(StringEnumConverter))]
+    public enum ResourceType
+    {
+        /// <summary>
+        /// The resource that's esigned is of type component.
+        /// </summary>
+        Component,
+
+        /// <summary>
+        /// The resource that's esigned is of type review.
+        /// </summary>
+        Review,
```

Comment:
Are you intending to sign a review and translation job?

Reply:
Not sure at this point, my thought was this would give us a chance to expand on to other artifacts as well. The changes I make now would only use "Component" but this would leave room for other major artifacts of Inspire to be esigned as well if needed.